### ISSUE TITLE
Choosing Node path when running SkyPortal

### ISSUE THREAD

**Issue by Theodlz** [skyportal/skyportal#2844]
HIi, i'm trying to deploy SkyPortal on a VM where the system is separate from a permanent disk that contains skyportal, python virtualenv (using anaconda), and node.js.

Current behavior:
When using make run, it uses the version of node.js from the system (so the one in the PATH)

Expected behavior:
When using make run, i can add an argument to choose the path to the node.js directory that suits my need (for example here for me being like /permanentdisk/node/...)

OR

I can specify the path to the node installation of my choice by adding it in the config.yaml file, so when using make run, it uses that node.js installation

**Comment by stefanv** [comment]
Why not just add the node of your choice to the system path?

**Comment by Theodlz** [comment]
The goal here is to meet the following requirements: as the system restarts it "looses" everything. It's a brand new system every time (it uses the same image that we built in advance), which is why we have a permanent disk that contains skyportal, python virtualenv and node. So we thought that being able to specify the node path, the same as for python (i think its already possible to specify the python path no ?), would be useful in that case, rather than having to add it to the path manually after reboot and before starting SkyPortal.

But,i know that it is not a priority for SkyPortal to add this kind of things, so if you don't think anyone has the time to work on it or that it won't be added for another reason, you can tell me and i'll just tell my colleagues that we should just do something else instead (on our side, rather than modifying SkyPortal).

**Comment by Theodlz** [comment]
Thanks for your answer Stefan. It seems that the people in charge of setting up the VM where i work changed their mind and just did that or something similar. So it's fixed !

Have a nice day.
